test/fit_tests/deploy/rackhd_package_install.py

```diff
@@ -0,0 +1,210 @@
+                    "dhcpGateway": "172.31.128.1",
+                    "dhcpProxyBindAddress": "172.31.128.1",
+                    "dhcpProxyBindPort": 4011,
+                    "dhcpSubnetMask": "255.255.252.0",
+                    "gatewayaddr": "172.31.128.1",
+                    "httpEndpoints": [
+                        {
+                            "address": "0.0.0.0",
+                            "port": fit_common.GLOBAL_CONFIG['ports']['http'],
+                            "httpsEnabled": False,
+                            "proxiesEnabled": True,
+                            "authEnabled": False,
+                            "routers": "northbound-api-router"
+                        },
+                        {
+                            "address": "0.0.0.0",
+                            "port": fit_common.GLOBAL_CONFIG['ports']['https'],
+                            "httpsEnabled": True,
+                            "proxiesEnabled": True,
+                            "authEnabled": True,
+                            "routers": "northbound-api-router"
+                        },
+                        {
+                            "address": "172.31.128.1",
+                            "port": 9080,
+                            "httpsEnabled": False,
+                            "proxiesEnabled": True,
+                            "authEnabled": False,
+                            "routers": "southbound-api-router"
+                        }
+                    ],
+                    "httpDocsRoot": "./build/apidoc",
+                    "httpFileServiceRoot": "./static/files",
+                    "httpFileServiceType": "FileSystem",
+                    "httpProxies": [{
+                        "localPath": "/mirror",
+                        "remotePath": "/",
+                        "server": fit_common.GLOBAL_CONFIG['repos']['mirror']
+                    }],
+                    "httpStaticRoot": "/opt/monorail/static/http",
+                    "minLogLevel": 3,
+                    "authUsername": "admin",
+                    "authPasswordHash": "KcBN9YobNV0wdux8h0fKNqi4uoKCgGl/j8c6YGlG7iA0PB3P9ojbmANGhDlcSBE0iOTIsYsGbtSsbqP4wvsVcw==",
+                    "authPasswordSalt": "zlxkgxjvcFwm0M8sWaGojh25qNYO8tuNWUMN4xKPH93PidwkCAvaX2JItLA3p7BSCWIzkw4GwWuezoMvKf3UXg==",
+                    "authTokenSecret": "RackHDRocks!",
+                    "authTokenExpireIn": 86400,
+                    "mongo": "mongodb://localhost/pxe",
+                    "sharedKey": "qxfO2D3tIJsZACu7UA6Fbw0avowo8r79ALzn+WeuC8M=",
+                    "statsd": "127.0.0.1:8125",
+                    "subnetmask": "255.255.252.0",
+                    "syslogBindAddress": "172.31.128.1",
+                    "syslogBindPort": 514,
+                    "tftpBindAddress": "172.31.128.1",
+                    "tftpBindPort": 69,
+                    "tftpRoot": "./static/tftp",
+                    "minLogLevel": 2
```

Comment:
Duplicated `minLogLevel` with line 139. For testing, I suggest setting the minLogLevel to zero, so more data will be printed.